TestApps/FileSystemNamedCollection.swift

```diff
@@ -0,0 +1,139 @@
+//
```

Comment:
Can you switch it to block comment to make it consistent with other files?

Reply:
Updated the template